sdk/textanalytics/azure-ai-textanalytics/samples/sample_invoke.py

```diff
@@ -0,0 +1,64 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from azure.core.pipeline.transport import HttpRequest, HttpResponse
+from azure.core.exceptions import HttpResponseError
+import json
+import os
+
+
+class InvokeSample(object):
+
+    def invoke(self):
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.textanalytics import TextAnalyticsClient
+
+        endpoint = os.environ["AZURE_TEXT_ANALYTICS_ENDPOINT"]
+        key = os.environ["AZURE_TEXT_ANALYTICS_KEY"]
+
+        text_analytics_client = TextAnalyticsClient(
+            endpoint=endpoint,
+            credential=AzureKeyCredential(key),
+            logging_enable=True
+        )
+
+        documents = [
+            """
+            The concierge Paulette was extremely helpful. Sadly when we arrived the elevator was broken, but with Paulette's help we barely noticed this inconvenience.
+            She arranged for our baggage to be brought up to our room with no extra charge and gave us a free meal to refurbish all of the calories we lost from
+            walking up the stairs :). Can't say enough good things about my experience!
+            """,
+            """
+            最近由于工作压力太大，我们决定去富酒店度假。那儿的温泉实在太舒服了，我跟我丈夫都完全恢复了工作前的青春精神！加油！
+            """
+        ]
+        documents = [{"id": str(idx), "text": doc} for idx, doc in enumerate(documents)]
+
+        request = HttpRequest("POST", "/languages",
+            headers={
+                'Content-Type': 'application/json'
+            },
+            data=json.dumps({
```

Comment:
Another discussion point; accept a `dict` or expect the caller to provide a serialized body....

Reply:
Let me add a second sample with the "set_json_body" that allows that

Reply:
Side note, as discussed offline, request would make it explicit with a "json" kwarg:
https://2.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests